### DISCUSSION TITLE
Vacuum Database

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1541] [Q&A]
Before using Vacuum Database the whole Trillium folder was a little above 800MB
After it went to 1.11GB
Is that normal? I thought Vacuum would actually reduce the database size.

Anyway, I guess the reason the Trillium folder already big is that I have a lot of images. Is there a way to remove unused ones?

**Comment by zadam** [comment]
That is quite strange. VACUUM should reduce the database size, definitely not increase it like that.

In the "Statistics" in the demo document there's a List of largest notes where you can see what takes most space. 

> Is there a way to remove unused ones?

No. Or perhaps it's better to say there's no concept of "unused image". Image can exist on its own in the hierarchy without being linked from anywhere.

Note also that when you delete a note, it's soft deleted at first and will take (by default) 7 days until it's hard deleted.

**Reply by hokutonanto** [reply]
> Note also that when you delete a note, it's soft deleted at first and will take (by default) 7 days until it's hard deleted.

Thanks, I didn't know that.
I'm going through the documentation and finally getting a better understanding of how Trillium works.
